Resources/Prototypes/Catalog/Fills/Boxes/service.yml

```diff
@@ -11,4 +11,26 @@
   - type: Sprite
     layers:
       - state: box
-      - state: flashbang
\ No newline at end of file
+      - state: flashbang
+
+- type: entity
+  name: AgriChem Is Fun! kit
```

Comment:
maybe change it to `"AgriChem Is Fun" kit`? and the same with manual `"AgriChem Is Fun" manual`

Reply:
Apparently you can only use quotes in the name if they're not the first character, so I've changed them to `NT "AgriChem is Fun" kit`